src/test_l1_plat_power.c

```diff
@@ -227,8 +229,10 @@ void test_l1_plat_power_negative_PLAT_TERM (void)
 	pmStatus_t ret;
 
     // Variation/Step 01: Call PLAT_TERM() - close interface without initializing
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     ret = PLAT_TERM();
     UT_ASSERT_EQUAL_FATAL(ret, PWRMGR_NOT_INITIALIZED); // Ensure the returned value is PWRMGR_NOT_INITIALIZED
```

Comment:
For  the enhance error codes NOT_INITIALISED or NOT_TERMINATED, most of the cases still return a different error.  This should be handled.
Handle in whole document

Reply:
Please update if this changes are done..